src/main/app/case/CosApiClient.test.ts

```diff
@@ -195,7 +195,7 @@ describe('CosApiClient', () => {
       'C100' as CaseType,
       'hearingNeeds' as CaseEvent
     );
-    expect(actual).toEqual(response);
+    expect(actual).toEqual({ id: '200', state: 'SUCCESS', hearingCollection: {} });
```

Comment:
what is the relevance of this one? is it because of the API change in the backend where we are getting hearing as an extra response

Reply:
yes that's correct.